easy_my_coop/models/operation_request.py

```diff
@@ -282,23 +282,25 @@ def validate(self):
                     )
                 )
 
-    def get_share_trans_mail_template(self):
+    def _get_share_transfert_mail_template(self):
```

Comment:
why is there a separate method for each type of message, while the contents are basically the same? it could be one method, used with arguments. the correspondence between all of the data (type of data, boolean field that controls whether to send, and template) could be defined as "constants".